patchwork-extensions-shearing/src/main/java/com/patchworkmc/mixin/extensions/shearing/MixinDispenserBehavior.java

```diff
@@ -47,7 +47,7 @@
  */
 @Mixin(targets = "net/minecraft/block/dispenser/DispenserBehavior$13")
 public class MixinDispenserBehavior extends FallibleItemDispenserBehavior {
-	private static final String DISPENSE_SILENTLY = "net/minecraft/block/dispenser/ItemDispenserBehavior.dispenseSilently(Lnet/minecraft/util/math/BlockPointer;Lnet/minecraft/item/ItemStack;)Lnet/minecraft/item/ItemStack;";
+	private static final String DISPENSE_SILENTLY = "dispenseSilently";
```

Comment:
I pushed the fix for this, this is no longer needed.

Reply:
will remove this change when i merge into upstream